core/src/utils/input-masking/utils/get-not-empty-selection.spec.ts

```diff
@@ -0,0 +1,73 @@
+import type { ElementState } from '../types/mask-interface';
+
+import { getNotEmptySelection } from './get-not-empty-selection';
+
+describe('getNotEmptySelection', () => {
+  it('should return the same selection when selection positions are not equal', () => {
+    const elementStateStub: ElementState = {
+      value: 'testValue',
+      selection: [1, 3],
+    };
+
+    expect(getNotEmptySelection(elementStateStub, true)).toEqual(elementStateStub.selection);
+    expect(getNotEmptySelection(elementStateStub, false)).toEqual(elementStateStub.selection);
+  });
+
+  describe('backward direction', () => {
+    it('should decrease by one start position value', () => {
+      const elementStateStub: ElementState = {
+        value: 'testValue',
+        selection: [4, 4],
+      };
+
+      expect(getNotEmptySelection(elementStateStub, false)).toEqual([3, 4]);
+    });
+
+    it('should not change everything when start value is 0', () => {
+      const elementStateStub: ElementState = {
+        value: 'testValue',
+        selection: [0, 4],
+      };
+
+      expect(getNotEmptySelection(elementStateStub, false)).toEqual([0, 4]);
+    });
+
+    it('should decrease by one start value', () => {
```

Comment:
This and the `'should decrease by one start position value'` test above seem to be testing the same thing.